packages/server/src/create-instance/inline.js

```diff
@@ -38,7 +28,12 @@ const createRenderStylesToString = (
   }
 
   if (globalStyles !== '') {
-    result = generateStyleTag(cssKey, globalIds, globalStyles, nonceString)
+    result = generateStyleTag(
+      cssKey,
+      globalIds.substring(1),
```

Comment:
I think this removes leading whitespace but I have not checked this. Either way - I've just hoisted this substring call from `generateStyleTag` so this should be totally equivalent so maybe it's not worth re-checking this.